<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>381</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Body Height:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arm Length:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Arm Velocity:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" alignment="Qt::AlignRight">
      <widget class="QLabel" name="armLengthLabel">
       <property name="text">
        <string>100 ft</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignRight">
      <widget class="QLabel" name="bodyHeightLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>212 ft</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2" alignment="Qt::AlignRight">
      <widget class="QLabel" name="armVelocityLabel">
       <property name="text">
        <string>100 mph</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QSlider" name="armLengthSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSlider" name="bodyHeightSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>212</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QSlider" name="armVelocitySlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="body">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>71</width>
      <height>212</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="arm">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrajectory">
    <property name="title">
     <string>Trajectory</string>
    </property>
   </widget>
   <addaction name="menuTrajectory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
